Modual/getBooks.js

```diff
@@ -1,6 +1,6 @@
 'use strict'
 
-const Book = require('../models/book.js')
+const Book = require('../models/books.js')
 
 async function grabBooks (request, response){
```

Comment:
I see you guys are well on your way, so nice work!

Feedback here: The file is named `getBooks` so consider changing the name of the function to `getBooks` 👍